client/my-sites/checkout/upsell-nudge/plan-upgrade-upsell/index.jsx

```diff
@@ -117,7 +120,7 @@ export class PlanUpgradeUpsell extends PureComponent {
 						</p>
 						<p>
 							{ translate(
-								"That's exactly why we've partnered with some of the world's greatest designers to offer nearly 200 high-end designs that you can use to make your site look incredible."
+								"That's exactly why we've partnered with some of the world's greatest designers to offer over 250 high-end designs that you can use to make your site look incredible."
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.